sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/TextTranslationClientBuilder.java

```diff
@@ -293,6 +295,19 @@ public TextTranslationClientBuilder resourceId(String resourceId) {
         return this;
     }
 
+    /**
+     * Sets the Authentication Scope used to authorize requests sent to the service.
+     *
+     * @param scope Token Scope.
+     * @return The updated {@link TextTranslationClientBuilder} object.
+     * @throws NullPointerException If {@code scope} is null.
+     */
+    public TextTranslationClientBuilder scope(String scope) {
```

Comment:
Is this the auth audience for supporting sovereign clouds? If so, we should use the existing pattern - see the following examples:
- [ContainerRegistry](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/ContainerRegistryAudience.java)
- [DocumentAnalysis](https://github.com/Azure/azure-sdk-for-java/blob/153cd0c80ec88e8e1510d2e58b3b2df4ffdcaf94/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentAnalysisAudience.java#L4)

Reply:
Done